capa/features/extractors/dnfile/types.py

```diff
@@ -42,15 +42,18 @@ def __repr__(self):
         return str(self)
 
     @staticmethod
-    def format_name(class_: str, namespace: str = "", member: str = ""):
+    def format_name(class_: tuple, namespace: str = "", member: str = ""):
+        if len(class_) > 1:
+            class_ = "/".join(class_) #class_[0] + "." + "/".join(class_[1:])
+        else:
+            class_ = "".join(class_) #f"{class_[0]}.{class_[1]}"
```

Comment:
Where does the `.` get inserted for these `join` calls?

Reply:
Updated comment to proper description.